setup.py

```diff
@@ -5,8 +5,8 @@
 req_lines = [line.strip() for line in open(
     'requirements.txt').readlines()]
 install_reqs = list(filter(None, req_lines))
-if sys.version_info[:2] == (2, 6):
-    install_reqs.append('importlib>=1.0.3')
+if sys.platform == 'win32':
+    install_reqs.append('textfsm==0.4.1')
```

Comment:
Why did we remove **importlib** ? 

Kindly add the comment saying win32 supports **testfsm 0.4.1.**

Reply:
We dont support python 2.6 and importlib was explicitly needed for that version.
>=2.7 python got importlib by default